packages/localize/src/formatters/xlb.ts

```diff
@@ -70,30 +70,17 @@ class XlbFormatter implements Formatter {
    * Read translations from all XLB files on disk that match the configured glob
    * pattern.
    */
-  async readTranslations(): Promise<Array<Bundle>> {
-    const files = await new Promise<string[]>((resolve, reject) =>
-      glob(
-        this.xlbConfig.translationsGlob,
-        {cwd: this.config.baseDir, absolute: true},
-        (err, files) => {
-          if (err) {
-            reject(err);
-          } else {
-            resolve(files);
-          }
-        }
-      )
-    );
-    const bundles: Array<Promise<Bundle>> = [];
+  readTranslations(): Array<Bundle> {
```

Comment:
Consider leaving async to preserve the option in the future? Maybe someday FS latency will be high enough and node will be optimized enough that doing it async in parallel is faster?

Reply:
Yeah, TS proper is pretty committed to being sync, for performance reasons, but that becomes an ecosystem integration problem. Seems like the `@rollup/typescript-plugin` should allow asynchrony.